worker-api/src/main/java/com/hpe/caf/api/worker/QueueTaskMessage.java

```diff
@@ -84,11 +84,6 @@ public final class QueueTaskMessage
      */
     private TaskSourceInfo sourceInfo;
 
-    /**
```

Comment:
Leave these in, if we remove them we'll have to increase the version of the TaskMessage and have compatibility to think about